packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -719,10 +720,10 @@ contract Deploy is Deployer {
                 _maxGameDepth: cfg.faultGameMaxDepth(),
                 _gameDuration: Duration.wrap(uint64(cfg.faultGameMaxDuration())),
                 _vm: faultVm,
-                _l2oo: L2OutputOracle(mustGetAddress("L2OutputOracleProxy"))
+                _l2oo: L2OutputOracle(mustGetAddress("L2OutputOracleProxy")),
+                _blockOracle: new BlockOracle()
```

Comment:
We should have a step `deployBlockOracle`?

Reply:
Doing in another PR.